### PR TITLE
fixed null pointer

### PR DESCRIPTION
**Pull request by Neill83**
null pointer fix in google-map.html

### CODE REVIEW

File: google-map.html [hunk 1]
```diff
@@ -194,7 +194,10 @@
       },
 
       detached: function() {
-        this.marker.setMap(null);
+          if(this.marker)
```
Comment (ebidel) [inline google-map.html]: style: `if(this.marker) {`

**Comment by Neill83** [comment]
CLA completed

File: google-map.html [hunk 2]
```diff
@@ -194,7 +194,9 @@
       },
 
       detached: function() {
-        this.marker.setMap(null);
+          if(this.marker) {
```
Comment (ebidel) [inline google-map.html]: spacing is still off: `if (this.marker) {`

**Comment by ebidel** [comment]
LGTM

**Pull request merged**